multinode_integration_tests/src/masq_mock_node.rs

```diff
@@ -579,7 +579,19 @@ impl MASQMockNode {
     }
 
     fn make_node_args(node_addr: &NodeAddr) -> Vec<String> {
-        vec![format!("{}", node_addr)]
+        vec![Self::add_tcp_specifications(format!("{}", node_addr))]
```

Comment:
`format!("{}", node_addr)` is an unidiomatic way to say `node_addr.to_string()`.

Reply:
True, true...